### PR TITLE
7020/convert playwright docs

### PR DESCRIPTION
**Pull request by MarkComerford**
#### 📲 What

Converted playwright docs to asciidoc

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: asciidoc/getting_started/playwright/plugin-information.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+----
+npm install --save-dev @ensono-stacks/playwright@latest
+----
+
+== yarn
+
+[source]
+----
+yarn add --dev @ensono-stacks/playwright@latest
+----
```
Comment (leesaxby) [inline asciidoc/getting_started/playwright/plugin-information.adoc:31]: For the npm and yarn code blocks i've just done the following. Don't mind if we use this style, but good to be consistent.
```
----
npm install --save-dev @ensono-stacks/playwright@latest
----

or

----
yarn add --dev @ensono-stacks/playwright@latest
----
```

File: asciidoc/getting_started/playwright/visual-regression.adoc [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+[source,text]
+----
+.
+├── apps
+│   ├── <app-name>-e2e
+│   │   ├── src
+│   │   │   ├── playwright-visual-regression.spec.ts #Example visual test using Playwright
+----
+[NOTE]
```
Comment (leesaxby) [inline asciidoc/getting_started/playwright/visual-regression.adoc:64]: I've used
```
NOTE: Visit the link:../../testing/testing_in_nx/playwright_visual_testing.adoc[Playwright with visual comparisons] documentation for further details!
```
for notes, tips etc.

Not sure if ones a better choice than the other, but would be good to be consistent.

File: asciidoc/getting_started/playwright/plugin-information.adoc (hunk 1 above)
Comment (MarkComerford) [reply asciidoc/getting_started/playwright/plugin-information.adoc:31]: updated this

File: asciidoc/getting_started/playwright/visual-regression.adoc (hunk 2 above)
Comment (MarkComerford) [reply asciidoc/getting_started/playwright/visual-regression.adoc:64]: changed this in various places to be NOTE:

**Review by leesaxby — APPROVED** [review]

**Pull request merged**